<!DOCTYPE html>
<html>
<head>
	<title>Projects - Benjamin R. Jack</title>
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<meta charset="utf-8">
	<link href="http://www.benjack.io/favicon.ico" rel="shortcut icon" type="image/x-icon">
	<link rel="stylesheet" type="text/css" href="http://www.benjack.io/style.css">
	<link rel="stylesheet" type="text/css" href="http://www.benjack.io/syntax.css">
	<script>
	  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
	  ga('create', 'UA-64753931-1', 'auto');
	  ga('send', 'pageview');
	</script>
</head>
<body>
	<header>
		
		
		<div class="container" id="navigation">
			<h1 id="name"><a href="/">Benjamin R. Jack</a></h1>
			<ul>
				<li><a href="/index.html#section1" class="red">About</a></li>
				<li><a href="/blog.html" class="orange">Blog</a></li>
				<li><a href="/cv.html" class="green">CV</a></li>
				<li><a href="/research.html" class="blue_active">Projects</a></li>
			</ul>
		</div>
	</header>
	<div class="container">

<h2>Research</h2>

<h3>Mechanistic Models of Viruses</h3>
<p>Until recently, modeling transcription and translation at individual base-pair resolution has been limited, due to computational constraints, to relatively short sequences. I want to move beyond modeling these short sequences. Modeling transcription and translation across a whole virus is the next logical step. Many viruses have compact genomes, short life-cycles, and they come with a wealth of detailed experimental data. My goal is to construct base-pair resolution models of viral transcription and translation. Such a model could predict the effects of promoter knockouts, genome rearrangements, codon changes, and many other manipulations that are now possible thanks to advances in synthetic biology. I plan to constrain these models with experimental data from collaborators at UT Austin. Ultimately, my goal is to use these models to understand how basic transcriptional and translational processes drive viral fitness and adaptation.</p>

<h3>Protein Evolution</h3>
<p>In a given protein, different amino-acid sites evolve at different rates. Amino acids near a catalytic site tend to be more conserved than residues elsewhere in the protein, and the protein must fold into the correct form to perform its catalytic activity. It follows, then, that there must be some relationship between the three-dimensional structure of a protein and the rate at which different parts of the protein evolve. My goal is to uncover the relationship between protein structure and protein evolution. Improving our understanding of the physical constraints on protein evolution could give us broad insights into viral protein evolution, drug design, antibiotic resistance, and other areas of study.</p>

<h2>Software</h2>

<h3>ParseMSF</h3>
<p>ParseMSF is an R package for parsing ThermoFisher MSF files and estimating protein abundances. The package provides several R functions for inspecting ThermoFisher MSF files, a proprietery file format commonly used in mass spectrometry-based protein quantitation. These functions retrieve peptide information, map peptides within a parent protein sequence, and estimate protein abundances based on peak peptide areas. I developed this package to facilitate automatic processing and manipulation of proteomics data in R. The ParseMSF package is publicly <a href="https://cran.rstudio.com/web/packages/parsemsf/index.html">available on CRAN</a>.</p>

<h3>Evolutionary Failure Mode Calculator</h3>
<p>In synthetic biology, organisms are engineered to produce compounds that they would not otherwise produce. Production of these foreign compounds often has a high fitness cost, so engineered organisms may accumulate mutations that disable this production. In other words, the engineered organisms break. I worked with Jeff Barrick to construct the <a href="http://barricklab.org/efm">Evolutionary Failure Mode (EFM) Calculator</a>, a computational tool that addresses the evolutionary instability of engineered organisms. The EFM Calculator predicts hyper-mutable sites in a given DNA sequence. It allows researchers to test sequences <em>in silico</em> before placing them into organisms. The EFM Calculator began as a rotation project during my first few months of graduate school and I continue to improve the Calculator's predictions and interface.</p>



			<ul id="contact" class="foot">
				<li><a href="mailto:benjamin.r.jack@gmail.com?Subject=Hello%20Benjamin" id="email">benjamin.r.jack@gmail.com</a></li>
				<li><a href="https://twitter.com/benrjack" id="twitter">@benrjack</a></li>
				<li><a href="http://www.linkedin.com/in/benjaminrjack" id="linkedin">benjaminrjack</a></li>
				<li><a href="http://www.github.com/benjaminjack" id="github">benjaminjack</a></li>
			</ul>


	</div>

</body>
</html>
